stl/inc/tuple

```diff
@@ -153,6 +153,8 @@ struct _Ignore { // struct that ignores assignments
 
 _INLINE_VAR constexpr _Ignore ignore{};
 
+// Note: This file uses extra _STD qualification within enable_if_t in order to improve throughput for large tuples.

```

Comment:
I don't think this is sufficient explanation for someone who hasn't read the commit log to correctly maintain this optimization. How about:
```suggestion
// Note: This file uses extra _STD qualification for names that appear in the arguments of enable_if_t to improve throughput. Specifically, we qualify names which appear anywhere in the STL as members of some class - including injected-class-names! - that we know are not members of the class being defined. This avoids pointless class-member lookup for those names in this context.
```

Reply:
Thanks for thinking about long-term maintainability - I completely agree. I've committed this, appropriately wrapped, and moving "to improve throughput" to the beginning for clarity (otherwise it sounds like improving throughput might be the purpose of names appearing in arguments of `enable_if_t`). Also set you as the `--author` since you did all the work of writing this excellent comment :smile_cat: